src/CTA.js

```diff
@@ -641,8 +641,10 @@ class CTA {
     }
 }
 
+export const MODULE_NAME = "Custom-Token-Animations";
```

Comment:
Does this not cause namespace collisions with other modules

Reply:
Yes it can be happened, but in that case you can use some alia slike this:
import { MODULE_NAME  as CustomTokenAnimationsName } from './CTA.js' 
import { MODULE_NAME  as TargetEnchantments } from './Target.js'